packages/create-payload-app/src/utils/getLatestPackageVersion.ts

```diff
@@ -19,7 +19,7 @@ export async function getLatestPackageVersion({
 }) {
   try {
     const response = await fetch(`https://registry.npmjs.org/${packageName}`)
-    const data = await response.json()
+    const data = (await response.json()) as { 'dist-tags': { latest: string } }
```

Comment:
Comment: Node-native fetch has `unknown` as return type - thus now it requires these type assertions.